### DISCUSSION TITLE
tutorials/workshop-summit24-prereqs/08-w16-ios

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#167] [course feedback]
# tutorials/workshop-summit24-prereqs/08-w16-ios

https://www.apollographql.com/tutorials/workshop-summit24-prereqs/08-w16-ios

**Comment by DroidPulkit** [comment]
How do I do this workshop?

**Reply by lizhennessy** [reply]
Hi there @DroidPulkit! Unfortunately the workshops you see listed here were those we presented live at GraphQL Summit last week. If you're interested in an iOS course, however, I have good news! [This course](https://www.apollographql.com/tutorials/apollo-ios-swift-part1) is a great place to start. It will take you through the basics of GraphQL in an iOS environment using Swift. Then you can go further into mutations, pagination, and subscriptions in [Part 2](https://www.apollographql.com/tutorials/apollo-ios-swift-part2). Hopefully these are useful to you!
